doc/source/comparison_with_r.rst

```diff
@@ -530,6 +530,103 @@ For more details and examples see :ref:`categorical introduction <categorical>`
 :ref:`differences to R's factor <categorical.rfactor>`.
 
 
```

Comment:
add a section tag here, like: ``_compare_with_r.string`` (actually if you can add them some of the sub-sections would be great). you put right *after* the sub-section label.